schema/SCHEMA_QUDT-v2.1.ttl

```diff
@@ -142,6 +143,11 @@ qudt:BaseUnit
   rdfs:label "Base Unit" ;
   rdfs:subClassOf qudt:Unit ;
 .
+qudt:BigEndian
```

Comment:
Not all your new entities have
rdfs:isDefinedBy <http://qudt.org/2.1/schema/qudt> ;